### PR TITLE
Sepolia V2 mock addresses

### PR DESCRIPTION
**Pull request by ytham**
- Updates Sepolia V2 mock addresses

### CODE REVIEW

**Comment by yi-sun** [comment]
CI failing?

**Comment by ytham** [comment]
> CI failing?

i think i just need to merge rc2

**Comment by ytham** [comment]
@yi-sun that did it; CI passing now

File: src/shared/chainConfig/sepolia.ts [hunk 1]
```diff
@@ -100,8 +100,8 @@ export let versionOverrideSepoliaMock: any = {
   },
   v2: {
     Addresses: {
-      Axiom: "0xF57252Fc4ff36D8d10B0b83d8272020D2B8eDd55",
-      AxiomQuery: "0x56A380e544606ad713A8A19BAc1903aa7C2017FF",
+      Axiom: "0x69963768F8407dE501029680dE46945F838Fc98B",
```
Comment (yi-sun) [inline src/shared/chainConfig/sepolia.ts:103]: Doesn't need to block this merge, but probably safer to call these `AxiomV2CoreMock` and `AxiomV2QueryMock`

**Review by yi-sun — APPROVED** [review]
LGTM

**Pull request merged**